components/event-bus/internal/knative/subscription/controller/subscription/reconcile_test.go

```diff
@@ -426,8 +431,8 @@ func makeKnChannel(namespace string, name string, labels map[string]string) *mes
 	}
 }
 
-func makeKnChannelName(kySub *eventingv1alpha1.Subscription) string {
-	return eventBusUtil.GetKnativeChannelName(&kySub.SourceID, &kySub.EventType, defaultMaxChannelLength)
+func makeKnChannelNamePrefix(kySub *eventingv1alpha1.Subscription) string {
```

Comment:
I'm a bit puzzled about the func name. It says it makes something with a prefix but returns another something called EventType. One of those are off.

Reply:
so initially the function name was *makeKnChannelName* this function was returning a derived name for channel. I've renamed it to **makeKnChannelNamePrefix** so it doesn't means make something with prefix. it just means make a channel name prefix. and we create prefix via Event-type parameter of kyma subscription.

Reply:
OK I get it now. It's an accessor with an obscure :)
I would argue that it introduces unnecessary obscurity but I won't insist.

Reply:
yea, i agree, but everywhere the accessors in this test start with *make* i don't think refactoring all of them is a good idea for this PR, whose intention is to get rid of dead code and fix tests based on that.